# Currently can take only one valid value: "einstein"
string robotname
---
string[] exprnames
